modules/uhttp/filterchain_builder.go

```diff
@@ -52,7 +53,15 @@ type filterChainBuilder struct {
 
 func defaultFilterChainBuilder(host service.Host) filterChainBuilder {
 	fcb := newFilterChainBuilder(host)
-	return fcb.AddFilters(contextFilter(host), panicFilter(host), tracingServerFilter(host), authorizationFilter(host))
+	scope := host.Metrics().Tagged(modules.HTTPTags)
+	return fcb.AddFilters(
+		panicFilter{scope.Counter("panic")},
+		metricsFilter{scope},
+		tracingServerFilter{scope},
+		authorizationFilter{
+			authClient:  host.AuthClient(),
+			authCounter: scope.Counter("auth.fail"),
```

Comment:
I think I'll take the fall for that, I suggested to use the dot.

The problem with doing two scopes in a row is that there is a lot of allocations and copying happening under the hood.

We should consider making it easier to create subscopes within tally, perhaps taking in a variadic list rather than a single string.